handlers/yo/yo.go

```diff
@@ -3,3 +3,253 @@ package yo
 /*
 GET /handlers/yo/received/uuid?account=12345&dest=8500&message=Msg&sender=256778021111
 */
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/nyaruka/courier"
+	"github.com/nyaruka/courier/handlers"
+	"github.com/nyaruka/courier/utils"
+	"github.com/pkg/errors"
+)
+
+var sendURL1 = "http://smgw1.yo.co.ug:9100/sendsms"
+var sendURL2 = "http://41.220.12.201:9100/sendsms"
+var sendURL3 = "http://164.40.148.210:9100/sendsms"
+
+func init() {
+	courier.RegisterHandler(NewHandler())
+}
+
+type handler struct {
+	handlers.BaseHandler
+}
+
+// NewHandler returns a new Yo! handler
+func NewHandler() courier.ChannelHandler {
+	return &handler{handlers.NewBaseHandler(courier.ChannelType("YO"), "YO!")}
+}
+
+// Initialize is called by the engine once everything is loaded
+func (h *handler) Initialize(s courier.Server) error {
+	h.SetServer(s)
+	s.AddReceiveMsgRoute(h, "POST", "receive", h.ReceiveMessage)
```

Comment:
Check error status, lets only register what these guys currently use. (look at our logs to figure out)